source/_components/sensor.swiss_hydrological_data.markdown

```diff
@@ -44,24 +46,46 @@ name:
   required: false
   type: string
   default: 
+measurements:
+  description: list of measurements you want to use.
+  required: true
+  type: list
+  deafult:
 {% endconfiguration %}
 
-The hydrological measurings are coming from the [Swiss Federal Office for the Environment (Bundesamt für Umwelt - Abt. Hydrologie)](http://www.hydrodaten.admin.ch) and are updated almost in real-time.
+Valid measurement values are:
 
-## {% linkable_title Example %}
+- temperature
```

Comment:
Move this section into the `keys` second of the `measurements` configuration value.